source/functions/BEMIO/readCAPYTAINE.m

```diff
@@ -505,6 +505,34 @@
 
 waitbar(8/8);
 
+%% Catch and remove nan values
+lowFrequencyNanMask = squeeze(all(isnan(hydro.A),[1 2])) | ...
```

Comment:
At this point the coefficients have dimensions of DOF, DOF (or size 1 for excitation), n frequencies.
So this line checks that a each frequency for data that is entirely NaN. If this happens in any of the hydrodynamic coefficients, those frequencies are removed.